packages/terra-popup/src/Popup.jsx

```diff
@@ -40,24 +56,30 @@ const propTypes = {
    */
   classNameOverlay: PropTypes.string,
   /**
-   * Attachment point for the popup, this will be mirrored to the target.
+   * Attachment point for the popup, this will be mirrored to the target. Options include: 'top left',
+   * 'top center', 'top right', 'middle left', 'middle center', 'middle right', 'bottom left',
+   * 'bottom center', or 'bottom right'.
    */
-  contentAttachment: PropTypes.oneOf(TetherComponent.attachmentPositions),
+  contentAttachment: PropTypes.oneOf(ATTACHMENT_POSITIONS),
   /**
    * A string representation of the height in px, limited to:
-   * 40, 80, 120, 160, 240, 320, 400, 480, 560, 640, 720, 800, 880
+   * 40, 80, 120, 160, 240, 320, 400, 480, 560, 640, 720, 800, 880 or dynamic.
    */
   contentHeight: PropTypes.oneOf(Object.keys(PopupHeights)),
   /**
    * A string representation of the width in px, limited to:
-   * 160, 240, 320, 640, 960, 1280, 1760
+   * 160, 240, 320, 640, 960, 1280, 1760 or dynamic.
```

Comment:
What are your thoughts on using "auto" in place of dynamic? It feels like it matches closer to the expected css behavior of height / width.

Reply:
Any other votes on this one?

Reply:
I like auto

Reply:
auto +1